hashtagsv2/hashtags/forms.py

```diff
@@ -14,6 +15,12 @@ class SearchForm(forms.Form):
 		widget=forms.TextInput(attrs={'placeholder': 'en.wikisource.org',
 			'style': 'width:100%'}))
 
+	search_type = forms.CharField(label="Type:", required=False,
```

Comment:
It's a small thing but could we be consistent about the order of these parameters? Below we start with required, then label :)